contracts/governance/Staking/Staking.sol

```diff
@@ -298,21 +298,113 @@ contract Staking is
     }
 
     /**
-     * @notice Withdraw tokens for vesting contract.
-     * @param vesting The address of Vesting contract.
+     * @notice Withdraw the given amount of tokens.
+     * @param amount The number of tokens to withdraw.
+     * @param until The date until which the tokens were staked.
      * @param receiver The receiver of the tokens. If not specified, send to the msg.sender
-     * @dev Can be invoked only by whitelisted contract passed to governanceWithdrawVesting.
+     * @param vestingConfig The vesting config.
```

Comment:
pls add details on the vesting config struct internals
else need to jump to the struct to figure out what this struct is about

Reply:
updated